scripts/purge-deployments.sh

```diff
@@ -11,11 +11,11 @@ else
   exit 1
 fi
 
-echo "Warning: this action will completely remove all the kubernetes deployments of ${current^^}. It cannot be undone!"
+echo "Warning: this action will completely remove all the kubernetes deployments of ${current}. It cannot be undone!"
 
 read -p 'Are you sure you want to remove all the kubernetes deployments (y/N)? ' yes_no
 
-if [[ $yes_no = 'y' || $yes_no = 'Y' ]]
+if [ $yes_no = 'y' -o $yes_no = 'Y' ];
```

Comment:
You're now using the `test` built-in instead of `[[`. `[[` has fewer surprises whereas `[` might not work in the way you expect sometimes -- just something to be aware of.

Reply:
After testing on both our toolbox pod and our developer macs, I am fully satisfied that the IF statements work as expected in every case.